code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -739,6 +743,91 @@
 	if(..())
 		visible_message("[src] safeties are locked on.")
 
+/mob/living/simple_animal/bot/mulebot/paranormal//allows ghosts only unless hacked to actually be useful
+	name = "paranormal MULEbot"
+	desc = "A Multiple Utility Load Effector bot. It only seems to accept paranormal forces, and for this reason is fucking useless."
+	icon_state = "paranormalmulebot0"
+	base_icon = "paranormalmulebot"
+	var/static/mutable_appearance/ghost_overlay
+	var/ghost_rider = FALSE
+
+/mob/living/simple_animal/bot/mulebot/paranormal/MouseDrop_T(atom/movable/AM, mob/user)
+	var/mob/living/L = user
+
+	if(user.incapacitated() || (istype(L) && !(L.mobility_flags & MOBILITY_STAND)))
+		return
+
+	if(!istype(AM) || iscameramob(AM) || istype(AM, /obj/effect/dummy/phased_mob)) //allows ghosts!
+		return
+
+	load(AM)
+
+/mob/living/simple_animal/bot/mulebot/paranormal/load(atom/movable/AM)
+	if(load ||  AM.anchored)
+		return
+
+	if(!isturf(AM.loc)) //To prevent the loading from stuff from someone's inventory or screen icons.
+		return
+
+	if(!istype(AM, /mob/dead/observer) && !wires.is_cut(WIRE_LOADCHECK))
+		buzz(SIGH)
+		return	// if not hacked, only allow ghosts to be loaded
+
+	var/obj/structure/closet/crate/CRATE
+	if(istype(AM, /obj/structure/closet/crate))
+		CRATE = AM
+	if(CRATE) // if it's a crate, close before loading
+		CRATE.close()
+
+	if(isobj(AM))
+		var/obj/O = AM
+		if(O.has_buckled_mobs() || (locate(/mob) in AM)) //can't load non crates objects with mobs buckled to it or inside it.
+			buzz(SIGH)
+			return
+
+	if(isliving(AM))
+		if(!load_mob(AM))
+			return
+	else
+		AM.forceMove(src)
+
+	load = AM
+	mode = BOT_IDLE
+	update_icon()
+	if(istype(AM, /mob/dead/observer))
+		ghost_rider = TRUE
+		RegisterSignal(AM, COMSIG_MOVABLE_MOVED, .proc/ghostmoved)
+
+/mob/living/simple_animal/bot/mulebot/paranormal/update_icon()
+	if(load && isobserver(load) && isnull(ghost_overlay))//there are issues with adding a ghost as an overlay, and this prevents metagaming to see who is dead
+		ghost_rider = TRUE
+		visible_message("<span class='warning'>A ghostly figure appears on [src]!</span>")
+		ghost_overlay = ghost_overlay || mutable_appearance('icons/mob/mob.dmi')
```

Comment:
braintoot: does this not return a T/F?

Reply:
I actually have no idea how mutable appearances work....

Reply:
this is fine, i just braintooted